public/pages/Dashboard/utils.ts

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { AD_DOC_FIELDS, SORT_DIRECTION } from '../../../server/utils/constants';
+
+/**
+ * Get the recent anomaly result query for the last timeRange period(Date-Math)
+ * Given timeRange is 24h, it return a query which is used to get anomaly result
+ * within last 24 hours
+ * @param  {[string]} timeRange [last time period which query is for]
+ * @returns query which is used to get anomaly result for the last timeRange period
+ */
+export const buildGetRecentAnomalyResultQuery = (timeRange: string) => {
+  return {
+    range: {
+      [AD_DOC_FIELDS.DATA_START_TIME]: {
+        gte: 'now-' + timeRange,
+      },
+    },
+    size: 30,
+    sortField: AD_DOC_FIELDS.DATA_START_TIME,
+    from: 0,
+    sortDirection: SORT_DIRECTION.DESC,
+  };
+};
+
+export type RgbColor = [number, number, number, number?];
+export const rgbColors: RgbColor[] = [
```

Comment:
Question: Why do we need this ?

Reply:
The sunburst chart of EUI I am using requires us to manually provide color palette: https://elastic.github.io/elastic-charts/?path=/story/sunburst--sunburst-with-two-layers

Reply:
As discussed offline, since there is no good way for us to leverage utils from elastic chart, our own implementation is the only solution for now.